### PR TITLE
Update climate.generic_thermostat.markdown

### PR DESCRIPTION
**Pull request by akking**
Update doc corresponds to the source code change

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#14427

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/climate.generic_thermostat.markdown [hunk 1]
```diff
@@ -14,26 +14,28 @@ ha_iot_class: "Local Polling"
...
+The `generic_thermostat` climate platform is a thermostat implemented in Home Assistant. It uses a sensor and a switch connected to a heater or air conditioning under the hood. The available operation modes are based on if the `heater` or `ac` are set in the config. When in heater mode, if the measured temperature is cooler then the target temperature, the heater will be turned on and turned off when the required temperature is reached. When in air conditioning mode, if the measured temperature is hotter then the target temperature, the air conditioning will be turned on and turned off when required temperature is reached.
 
 ```yaml
 # Example configuration.yaml entry
 climate:
   - platform: generic_thermostat
     name: Study
     heater: switch.study_heater
+    ac: switch.study_ac
```
Comment (fabaff) [inline source/_components/climate.generic_thermostat.markdown:25]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/climate.generic_thermostat.markdown [hunk 2]
```diff
@@ -14,26 +14,28 @@ ha_iot_class: "Local Polling"
...
 - **name** (*Required*): Name of thermostat
-- **heater** (*Required*): `entity_id` for heater switch, must be a toggle device. Becomes air conditioning switch when `ac_mode` is set to `True`
+- **heater** (*Optional*): `entity_id` for heater switch, must be a toggle device.
+- **ac** (*Optional*): `entity_id` for air conditioning switch, must be a toggle device.
 - **target_sensor** (*Required*): `entity_id` for a temperature sensor, target_sensor.state must be temperature.
 - **min_temp** (*Optional*): Set minimum set point available (default: 7)
 - **max_temp** (*Optional*): Set maximum set point available (default: 35)
 - **target_temp** (*Optional*): Set initial target temperature. Failure to set this variable will result in target temperature being set to null on startup. As of version 0.59, it will retain the target temperature set before restart if available.
-- **ac_mode** (*Optional*): Set the switch specified in the *heater* option to be treated as a cooling device instead of a heating device.
+- **ac_mode** (*Optional*): Set true if you want set the initial mode to ac mode.
```
Comment (fabaff) [inline source/_components/climate.generic_thermostat.markdown:38]: There is no "initial mode" as you can't change the mode during runtime.

File: source/_components/climate.generic_thermostat.markdown [hunk 3]
```diff
@@ -14,26 +14,28 @@ ha_iot_class: "Local Polling"
 ---
 
 
-The `generic_thermostat` climate platform is a thermostat implemented in Home Assistant. It uses a sensor and a switch connected to a heater or air conditioning under the hood. When in heater mode, if the measured temperature is cooler then the target temperature, the heater will be turned on and turned off when the required temperature is reached. When in air conditioning mode, if the measured temperature is hotter then the target temperature, the air conditioning will be turned on and turned off when required temperature is reached.
+The `generic_thermostat` climate platform is a thermostat implemented in Home Assistant. It uses a sensor and a switch connected to a heater or air conditioning under the hood. The available operation modes are based on if the `heater` or `ac` are set in the config. When in heater mode, if the measured temperature is cooler then the target temperature, the heater will be turned on and turned off when the required temperature is reached. When in air conditioning mode, if the measured temperature is hotter then the target temperature, the air conditioning will be turned on and turned off when required temperature is reached.
```
Comment (frenck) [inline source/_components/climate.generic_thermostat.markdown:17]: `when required` -> `when the required`

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed as well.
Feel free to re-open when ready.

**Pull request closed without merging**